docs/services/xks/operator-guide.md

```diff
@@ -3,6 +3,239 @@ id: operator-guide
 title: Operator Guide
 ---
 
-# Bootstrap
-# Add New Tenant
-# Clean Up
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Bootstrap
+
+## Add New Tenant
+
+When creating a new tenant there a number of for now manual processes to make.
+
+In this scenario we are assuming that you are using azure devops and that you have already created a project and organization.
+
+In your project lets call it project1 in the future manually import a repository.
+[https://github.com/XenitAB/azure-devops-templates](https://github.com/XenitAB/azure-devops-templates)
+
+### Import azure-devops-templates pipeline
+
+To make sure that the azure-devops-templates repo is up to date we have a automatic CI that fetches updates from upstream to your local azure devops clone.
+
+Go to pipelines -> New pipeline -> Azure Repos Git -> azure-devops-templates -> Existing Azure Pipelines YAML file
+
+Import the pipeline from the following path: /.ci/pipeline.yaml
+
+### Setup XKS
+
+In this case we will only setup a single XKS cluster in one environment, in our case dev.
+It's easy to add more environments when you have created your first one.
+
+At Xenit we are using terraform modules that we share [upstream](https://github.com/XenitAB/terraform-modules)
+
+To setup XKS we will utalize 3 modules:
+
+- governance-global
```

Comment:
regional?

Reply:
Yeah I thought of that as well.
Do we have some word for having 2 modules in 1 folder?
groups or something. My thught was that I wanted to have a easy connection to the developers file strucutre. 
But I should probably just go with 4 modules and explain how the folder strucutre looks like.